roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -17,7 +17,7 @@
     - {"path": "/var/lib/edpm-config/firewall", "owner": "root", "group": "root"}
     - {"path": "/etc/pki/libvirt", "owner": "root", "group": "root"}
     - {"path": "/etc/pki/libvirt/private", "owner": "root", "group": "root"}
-    - {"path": "/etc/pki/CA/libvirt", "owner": "root", "group": "root"}
+    - {"path": "/etc/pki/qemu", "owner": "root", "group": "{{ edpm_nova_libvirt_qemu_group | default('qemu') }}"}
```

Comment:
@gibizer is this correct? Not sure if it should just be hard-coded to qemu

Reply:
Based on the documentaton edpm_nova_libvirt_qemu_group (The vhost user socket directory group name.) the intention of this variable is different. So I would just hardcode qemu here.